grails-app/domain/org/pih/warehouse/inventory/OutboundStockMovementListItem.groovy

```diff
@@ -102,17 +103,17 @@ class OutboundStockMovementListItem implements Serializable, Validateable {
         statusSortOrder(nullable: true)
     }
 
-    def getDisplayStatus() {
-        StockMovementStatusHelper statusHelper = new StockMovementStatusHelper(
+    @Deprecated
```

Comment:
same here. Please document why we're deprecating and what we should use instead.